### PR TITLE
wip: added the basic template

### PR DESCRIPTION
**Pull request by dks-ar**
Created a basic template for pulumi typescript.

### CODE REVIEW

File: fetch/rest.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const request = require('request');
+export const req = request('http://www.google.com', function (error:any, response:any, body:any) {
+  console.log('error:', error); // Print the error if one occurred
+  console.log('statusCode:', response && response.statusCode); // Print the response status code if a response was received
+  console.log('body:', body); // Print the HTML for the Google homepage.
```
Comment (dhieru) [inline fetch/rest.ts]: @dks-ar lgtm, remove this line, body could be huge.

**Review by dhieru — CHANGES_REQUESTED** [review]
looks good to me, added one comment please fix that

File: fetch/rest.ts (hunk 1 above)
Comment (dks-ar) [reply fetch/rest.ts]: @dhieru thanks for the review, fixed in `318d5780`

**Review by dhieru — COMMENTED** [review]
@dks-ar  please squash  and commit

**Review by dhieru — APPROVED** [review]

**Pull request merged**